rules/when-to-use-anchoring/rule.md

```diff
@@ -0,0 +1,53 @@
+---
+type: rule
+title: Do you know when to use anchoring?
+uri: when-to-use-anchoring
+authors:
+  - title: Tanya Leahy
+    url: https://ssw.com.au/people/tanya-leahy
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  guid: 69b80ab3-e4fa-4624-93a5-3ec6c01141d8
+created: 2024-05-03T18:29:33.0000000Z
+related: 
+  - share-the-agenda
+  - stick-to-the-agenda-and-complete-the-meetings-goal
+  - anchoring-sales-strategy
+redirects: []
+  
+
+---
+Anchoring is a cognitive bias where individuals rely heavily on the first piece of information they encounter (the "anchor") when making decisions or judgments. This initial information then influences subsequent thinking and judgments, often leading to skewed conclusions or estimations. Anchoring can occur consciously or unconsciously and can significantly impact decision-making processes across various contexts, from negotiations and purchases to problem-solving and risk assessment.
+
+While anchoring can be a powerful technique in certain situations, it's essential to consider its potential drawbacks and the context in which it is used to achieve the most postitive outcome to a situation. During meetings, senior people should give their opinions last to avoid creating bias to foster better collaboration and discussion.
+
+<!--endintro-->
+
+`youtube: https://youtu.be/JL4OoKJyNrc`
+**Video: Jeff Bezos - Talking about bias in meetings**
+
+## When not to use anchoring in a meeting
+
+1. **Misleading Intentions:** Anchoring should not be used with the intention of misleading or manipulating others, as this can erode trust and undermine collaborative relationships in the workplace.
+2. **Ignoring True Value:** Overreliance on anchoring can detract from genuinely assessing the value or merits of alternatives, hindering creative problem-solving and innovation.
+3. **Rigidity in Negotiation:** Using anchoring excessively can create rigidity in negotiation dynamics, making it challenging to adapt to changing circumstances or explore mutually beneficial solutions.
+4. **Ethical Considerations:** Anchoring should be used ethically and transparently, avoiding tactics that exploit or disadvantage other parties involved in the workplace interaction.
+
+::: greybox
+At the start of a meeting, the CEO gives their opinion upfront on the topic to be discussed and creates bias amongst the team.
```

Comment:
@Tanya-Leahy 

The greybox here is missing the closing 
:::

- [ ] 1. Please fix
- [ ] 2. Do the same on the good example below

Reply:
Done